test/Vagrantfile

```diff
@@ -66,8 +67,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         target.ssh.password = "vagrant"
         target.vm.network "private_network", ip: "172.31.128.1", virtualbox__intnet: "closednet"
         target.ssh.forward_agent = true
+        target.vm.network "forwarded_port", guest: 8080, host: 8080
+        target.vm.network "forwarded_port", guest: 9080, host: 9080
+        target.vm.network "forwarded_port", guest: 9090, host: 9090
+        target.vm.network "forwarded_port", guest: 443, host: 443
         target.vm.provision "shell", inline: <<-SHELL
-            git clone https://github.com/RackHD/RackHD fit_tests
+            git clone https://github.com/RackHD/RackHD

```

Comment:
That is correct. The confusing part is that this is running locally on the server rather than from a third party machine. This download just runs the deployment stuff. Confused me for a bit too.